packages/lodestar/src/util/jwt.ts

```diff
@@ -0,0 +1,28 @@
+import {encode, decode, TAlgorithm} from "jwt-simple";
```

Comment:
You can move this utility to `packages/lodestar/src/eth1/provider/` since it's exclusive to that code.

Reply:
move the utility and its tests to suggested sections https://github.com/ChainSafe/lodestar/pull/3777/commits/8f8197d55d2021f04fa1927cd6bd76c9b4db1c5d